app/actors/hyrax/actors/featured_collection_actor.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Actors
+    # Removes featured collections if the collection is deleted or becomes private
+    class FeaturedCollectionActor < Hyrax::Actors::AbstractActor
```

Comment:
do collections go through the same actor stack as works do or is it a separate flow?

Reply:
I'm not _entirely_ sure if this answers what you are asking, but the structure is that there is both a `FeaturedCollectionActor` and a `FeaturedWorkActor`. Both of these actors are called in the `DefaultMiddlewareStack`

Reply:
So I think there is a separate flow.

Reply:
but you are adding it to the middleware stack for works.  That seems like a bug. lets pair up and investigate this before we merge